storage/innobase/include/srv0srv.h

```diff
@@ -305,7 +305,7 @@ extern uint	srv_flush_log_at_timeout;
 extern char	srv_use_global_flush_log_at_trx_commit;
 extern char	srv_adaptive_flushing;
 
-#ifdef WITH_INNODB_DISALLOW_WRITES
+#if defined(WITH_WSREP) AND defined(WITH_INNODB_DISALLOW_WRITES)
```

Comment:
I think this part can be removed. There is no srv_allow_writes_event variable anywhere :)